src/main/kotlin/org/jetbrains/research/ml/ast/transformations/PyElementGeneratorExt.kt

```diff
@@ -19,3 +20,16 @@ fun PyElementGenerator.createBoolLiteralExpression(value: Boolean): PyBoolLitera
 fun PyElementGenerator.createExpressionFromNumber(value: Number): PyExpression {
     return createExpressionFromText(defaultLanguageLevel, value.toString())
 }
+
+fun PyElementGenerator.createBinaryOperandList(operator: String, operands: List<PyExpression>): PyExpression {
+    require(operands.isNotEmpty()) { "operands list should not be empty" }
+    return operands
+        .map { PyUtils.braceExpression(it) }
+        .reduce { lhs, rhs -> createBinaryExpression(operator, lhs, rhs) }
+}
+
+fun PyElementGenerator.createPrefixExpression(operator: String, operand: PyExpression): PyPrefixExpression {
+    val prefixExpression = createExpressionFromText(defaultLanguageLevel, "${operator}1") as PyPrefixExpression
```

Comment:
Maybe should we send `-1` as an argument? I mean the `not` operator is also a prefix expression. But we should not add `1` if we want to create this one.

Reply:
An extra space is now added for this purpose when needed